tracer/test/Datadog.Trace.Tests/Telemetry/telemetry_app-started.json

```diff
@@ -66,6 +66,11 @@
           "message": "Invalid value"
         }
       }
-    ]
+    ],
+    "install_signature": {
+      "install_id": "68e75c48-57ca-4a12-adfc-575c4b05fcbe",
+      "install_type": "k8s_single_step",
+      "install_time": "1703188212"
```

Comment:
We should double-check that this definitely shouldn't be a number, because they use numbers for this in the tag example in the RFC (though I'm not sure that's possible).

To be clear, I realise in the spec shows this as a string currently, we should just double-check that's corretc and not a typo

Reply:
I confirm that [our backend expects install_time in the JSON](https://github.com/DataDog/dd-go/blob/instrumentation-telemetry/integration/22-12-23-codefreeze/trace/apps/tracer-telemetry-intake/telemetry-payload/app_started.go#L45-L49) to be a string and not a number and